src/main/resources/com/google/api/codegen/java/sample.snip

```diff
@@ -9,10 +9,12 @@
    *    instructions at https://github.com/google/google-api-java-client.
    *    On other build systems, you can add the jar files to your project from
    *    https://developers.google.com/resources/api-libraries/download/{@sampleClass.apiName}/{@sampleClass.apiVersion}/java
-   * 3. This sample uses Application Default Credentials for authentication.
-   *    If not already done, install the gcloud CLI from
-   *    https://cloud.google.com/sdk and run
-   *    `gcloud beta auth application-default login`
+   @if sampleClass.body.authType == "APPLICATION_DEFAULT_CREDENTIALS"
```

Comment:
(admittedly I did this to save some lines in the snippet file...)

but I did also think that the order made sense:
1: enable the api
2: download the client library
3: authenticate

i mean, it's also kind of arbitrary, but if i remember psychology correctly the first and last items of a list are best recalled, and they are the two most important items :)